docs/topics/proc_interacting-with-the-circuit-breaker-booster.adoc

```diff
@@ -1,16 +1,17 @@
 // This is a parameterized module. Parameters used:
 //
 //
-//  parameter-mission: id of the mission. This is used in anchors file imports.
-//  parameter-mission-name: used in the mission title
+//  parameter-example: id of the example. This is used in anchors file imports.
+//  parameter-example-name: used in the example title
 //
 // Rationale: This procedure is identical in all deployments.
 // TODO: Add conditional includes for Node.
 
 [#interacting-with-the-unmodified-booster-{context}]
-= Interacting with the unmodified {runtime} {parameter-mission-name} booster
+[id='interacting-with-the-unmodified-example_{context}']
+= Interacting with the unmodified {runtime} {parameter-example-name} example
 
-Once you have the {runtime} booster deployed, you have the following services running:
+Once you have the {runtime} example deployed, you have the following services running:
```

Comment:
Once you have the {runtime} example deployed--> After you have the {runtime} example application deployed